lib/wcc/contentful/simple_client/typhoeus_adapter.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+gem 'typhoeus'
+require 'typhoeus'
+
+class TyphoeusAdapter
+  def call(url, query, headers = {}, proxy = {})
+    raise NotImplementedError, 'Proxying Not Yet Implemented' if proxy[:host]
+
+    TyphoeusAdapter::Response.new(
+      Typhoeus.get(
```

Comment:
I guess we're always doing a `get`? Never a need to do a `delete` or `post`?

Reply:
Haven't had that use case yet